examples/meta/src/evaluation/clustering.sg

```diff
@@ -0,0 +1,33 @@
+Math:init_random(1)
+
+File f_feats_train = csv_file("../../data/optdigits_7class_feats.dat")
+File f_labels_train = csv_file("../../data/optdigits_7class_labels.dat")
+File f_labels_centroids = csv_file("../../data/9class_continuous_labels.dat")
+
+#![create_features]
+Features features_train = features(f_feats_train)
+Labels labels_train = labels(f_labels_train)
+#![create_features]
+
+#![run_clustering]
+Distance d = distance("EuclideanDistance", lhs=features_train, rhs=features_train)
+Machine kmeans = machine("KMeans", k=10, distance=d)
+kmeans.train()
+RealMatrix centers = kmeans.get_real_matrix("cluster_centers")
+#![run_clustering]
+
+#![assign_labels]
+Labels labels_centroids = labels(f_labels_centroids)
+Features fea_centroids = features(centers)
+Distance d2 = distance("EuclideanDistance", lhs=fea_centroids, rhs=fea_centroids)
+Machine knn = machine("KNN", k=1, distance=d2, labels=labels_centroids)
+knn.train()
+Labels gnd_hat = knn.apply(features_train)
+#![assign_labels]
+
+#![measure_accuracy]
+Evaluation accuracy_eval = evaluation("ClusteringAccuracy")
```

Comment:
Just remove this and store the cluster centres instead

Reply:
Isn't this unsupervised learning? You cluster using KMeans, and used KNN around these clusters to predict the labels, then match your made up labels with the original. Sorry for persisting, but I think it makes sense, but if you don't think so though I will remove the evaluation code.

Reply:
@karlnapf My bad :"D After writing the previous comment I realized that `k` should be equal to the number of labels, I will try it now.

Reply:
it didn't make a difference, I will drop accuracy